ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -381,12 +382,12 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/Flutter",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.analogio.coffeecard;
+				PRODUCT_BUNDLE_IDENTIFIER = DK.AnalogIO.DigitalCoffeeCard;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "cicd-iosdev";
+				PROVISIONING_PROFILE_SPECIFIER = "githubactions-dev";
 				SWIFT_OBJC_BRIDGING_HEADER = "Runner/Runner-Bridging-Header.h";
 				SWIFT_VERSION = 5.0;
-				TARGETED_DEVICE_FAMILY = 1;
+				TARGETED_DEVICE_FAMILY = "1,2";
```

Comment:
This means the app targets iPad. Is this intentional? (also on line 535, 570)

Reply:
Yes, it is even though it's a bit unfortunate. The old iOS app also targeted iPads which means that when we want to upgrade the existing app, we must target the same devices.